packages/analytics-remote-config/src/types.ts

```diff
@@ -6,16 +6,20 @@ export interface RemoteConfigAPIResponse<RemoteConfig extends { [key: string]: o
   };
 }
 
-export interface RemoteConfigFetch<T> {
+export interface BaseRemoteConfigFetch<T> {
   getRemoteConfig: <K extends keyof T>(
     configNamespace: string,
     key: K,
     sessionId?: number,
   ) => Promise<T[K] | undefined>;
 }
 
+export interface RemoteConfigFetch<T> extends BaseRemoteConfigFetch<T> {
+  fetchTime: number;
```

Comment:
Because fetch time is for network request. I isolate it so that `RemoteConfigIDBStore` doesn't need to have fetch time. `RemoteConfigIDBStore` is for indexedDB.